outputs.tf

```diff
@@ -12,3 +22,61 @@ output "storage_account_name" {
   description = "The name of the storage account."
   value       = local.storage_account_name
 }
+
+output "storage_account_primary_location" {
+  description = "The primary location of the storage account"
+  value       = one(module.storage[*].storage_account_primary_location)
+}
+
+output "storage_account_primary_blob_endpoint" {
+  description = "The endpoint URL for blob storage in the primary location."
+  value       = one(module.storage[*].storage_account_primary_blob_endpoint)
+}
+
+output "storage_account_primary_web_endpoint" {
+  description = "The endpoint URL for web storage in the primary location."
+  value       = one(module.storage[*].storage_account_primary_web_endpoint)
+}
+
+output "storage_account_primary_web_host" {
+  description = "The hostname with port if applicable for web storage in the primary location."
+  value       = one(module.storage[*].storage_account_primary_web_host)
+}
+
+output "storage_primary_connection_string" {
+  description = "The primary connection string for the storage account"
+  value       = one(module.storage[*].storage_primary_connection_string)
+  sensitive   = true
+}
+
+output "storage_primary_access_key" {
+  description = "The primary access key for the storage account"
+  value       = one(module.storage[*].storage_primary_access_key)
+  sensitive   = true
+}
+
+output "storage_secondary_access_key" {
```

Comment:
Some of the descriptions are without a dot and some of them are with a dot. My proposal is to unify that.